web/src/instrumentation.ts

```diff
@@ -1,7 +1,14 @@
 // See: https://vercel.com/docs/observability/otel-overview
 
 export async function register() {
-  if (process.env.NEXT_RUNTIME === "nodejs") {
+  // This variable is set in the .env file or environment variables
+  // Value is true if NEXT_PUBLIC_LANGFUSE_RUN_NEXT_INIT_ENABLED is "true" or undefined
+  const isInitLoadingEnabled =
+    process.env.NEXT_PUBLIC_LANGFUSE_RUN_NEXT_INIT_ENABLED !== undefined
```

Comment:
Suggestion

- I would not add another env (NEXT_PUBLIC_LANGFUSE_RUN_NEXT_INIT_ENABLED)
- We could rely on whether LANGFUSE_INIT_ORG_ID exists as it is the topmost check in the initalization file, probably worth adding a note to the initialization script that the condition needs to be updated if the init script changes
- LANGFUSE_INIT_ORG_ID does not exist on localhost usually but is sufficiently tested in ci as our ci pipeline depends on it

Reply:
@marcklingen In that case I'd only make the `initialize` script conditional. Trying the otel/datadog init to `LANGFUSE_INIT_ORG_ID` feels wrong and unintuitive.

Reply:
I fully agree here with @Steffen911 and hence added a new env variable.

Reply:
Sorry for the confusion, I remembered that you wanted to keep the datadog instrumentation at all times. With the timings below it makes sense to remove datadog as well.

Change looks good to me